mail_show_follower/models/mail_mail.py

```diff
@@ -0,0 +1,90 @@
+                    # internal partners
+                    user_partner_ids = (
+                        self.env["res.users"]
+                        .search(
+                            [
+                                ("active", "in", (True, False)),
+                                ("show_in_cc", "=", False),
+                            ]
+                        )
+                        .filtered(lambda x: group_portal not in x.groups_id)
+                        .mapped("partner_id")
+                        .ids
+                    )
+                    partners_len = len(
+                        partners_obj.filtered(
+                            lambda x: x.id not in user_partner_ids
+                            and (
+                                not x.user_ids
+                                or group_portal in x.user_ids.mapped("groups_id")
+                            )
+                        )
+                    )
+                    if partners_len > 1:
+                        # get partners
+                        cc_internal = True
+                        # else get company in object
+                        if hasattr(obj, "company_id") and obj.company_id:
+                            cc_internal = obj.company_id.show_internal_users_cc
+                        # get company in user
+                        elif mail.env and mail.env.user and mail.env.user.company_id:
+                            cc_internal = (
+                                self.env.user.company_id.show_internal_users_cc
+                            )
+                        if cc_internal:
+                            partners = partners_obj.filtered(
+                                lambda x: x.id not in user_partner_ids
+                                and (not x.user_ids or x.user_ids.show_in_cc)
+                            )
+                        else:
+                            partners = partners_obj.filtered(
+                                lambda x: x.id not in user_partner_ids
+                                and (
+                                    not x.user_ids
+                                    or group_portal in x.user_ids.mapped("groups_id")
+                                )
+                            )
+                        partners = partners.filtered(
+                            lambda x: not x.user_ids
+                            or x.user_ids  # otherwise, email is not sent
+                            and "email" in x.user_ids.mapped("notification_type")
+                        )
+                        # get names and emails
+                        final_cc = None
+                        mails = ""
+                        for p in partners:
+                            mails += "%s &lt;%s&gt;, " % (p.name, p.email)
+                        # join texts
+                        final_cc = plain_text % (mails[:-2])
```

Comment:
This can be replaced with: 
```
final_cc = plain_text % (", ".join("%s &lt;%s&gt;" % (p.name, p.email) for p in partners))
```

Correct?

Reply:
I applied the improvement. Thanks for the suggestion.